DllImportGenerator/DllImportGenerator/DllImportStub.cs

```diff
@@ -107,8 +108,11 @@ public class GeneratedDllImportData
         }
 
         public static DllImportStub Create(
```

Comment:
I'm not a fan of this. The mapping logistics are imposing a fair bit of data population I don't see aligning with the current design. If we were to start over here what would we do differently? Is there a minimum architecture change that would make this function not need to expand? Could we tag the `StubEnvironment` type with some mapper cache instead?

Reply:
I've put up an alternate proposal at #1083 that I think you'll find to be cleaner. Let me know what you think!